robotpy_installer/installer.py

```diff
@@ -137,7 +137,7 @@ def roborio_checks(
             "grep IMAGEVERSION /etc/natinst/share/scs_imagemetadata.ini",
         )
 
-    m = re.match(r'IMAGEVERSION = "FRC_roboRIO_(.*)"', result.strip())
+    m = re.match(r'IMAGEVERSION = "FRC_roboRIO(?:2|)_(.*)"', result.strip())
```

Comment:
This doesn't sit right with me. There could be a case in the future where the roborio1 and 2 versions are different? Also, probably want to change the log message below to 'RoboRio2 image version'...

Reply:
Can't test this out, but does it look better now?